charts/astronomer/templates/houston/api/houston-deployment.yaml

```diff
@@ -48,6 +48,11 @@ spec:
           resources:
 {{ toYaml .Values.houston.resources | indent 12 }}
           env:
+            - name: DATABASE_URL
+              valueFrom:
+                secretKeyRef:
+                  name: astronomer-bootstrap
```

Comment:
Can we use the secret name as "{{ template "houston.backendSecret" . }}" instead of hardcoding the name?